addons/account/wizard/accrued_orders.py

```diff
@@ -54,7 +54,7 @@ def _get_default_company(self):
     preview_data = fields.Text(compute='_compute_preview_data')
     display_amount = fields.Boolean(compute='_compute_display_amount')
 
-    @api.depends('date', 'amount')
+    @api.depends('date', 'amount', 'preview_data')
```

Comment:
In the compute method, shouldn't `self.preview_data` be `record.preview_data` instead?